doubles/allowance.py

```diff
@@ -179,6 +193,20 @@ def satisfy_exact_match(self, args, kwargs):
 
         return True
 
+    def satisfy_custom_matcher(self, args, kwargs):
+        """
+        Returns a boolean indicating whether or not the stub will accept the provided arguments.
```

Comment:
`Return`, and on the previous line. I know the rest of the codebase follow this other convention, but probably safer to use the more widely accepted pep257 one.